sdk/go/Makefile

```diff
@@ -41,10 +41,18 @@ GENERATED_OUTBOUND_REDIS = redis/outbound-redis.c redis/outbound-redis.h
 GENERATED_SPIN_REDIS     = redis/spin-redis.c redis/spin-redis.h
 GENERATED_KEY_VALUE      = key_value/key-value.c key_value/key-value.h
 
+SDK_VERSION_SOURCE_FILE  = sdk_version/sdk-version-go-original.c
+SDK_VERSION_DEST_FILES   = config/sdk-version-go.c http/sdk-version-go.c \
```

Comment:
This seems like the only fragile bit, in terms of new areas of functionality being added to the go SDK (e.g. `postgres/`, etc.)  I'm assuming it's our best option currently.  If so, what might our strategy be to ensure they are added to the list here?

Reply:
Good thinking.  I can do a bit of Makefile function magic to make it more robust.

Reply:
On second thought, I just added a (hopefully attention-grabbing) comment indicating what needs to be updated on the rare occasion we add a new directory to the Go SDK.